webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/DeleteUserDialog.tsx

```diff
@@ -25,19 +25,23 @@ import { useService } from '@cloudbeaver/core-di';
 import type { DialogComponent } from '@cloudbeaver/core-dialogs';
 import { NotificationService } from '@cloudbeaver/core-events';
 
+import { UsersTableOptionsPanelService } from '../UsersTable/UsersTableOptionsPanelService.js';
+
 interface IPayload {
   userId: string;
 }
 
 export const DeleteUserDialog: DialogComponent<IPayload> = function DeleteUserDialog(props) {
   const translate = useTranslate();
   const notificationService = useService(NotificationService);
+  const usersTableOptionsPanelService = useService(UsersTableOptionsPanelService);
   const usersResource = useResource(DeleteUserDialog, UsersResource, null);
```

Comment:
This is strange, you can use `useService(UsersResource);`

Reply:
Yea, that strange, we dont use loaded data anywhere in the component, changed it to useService